### DISCUSSION TITLE
.fda / .fds converter deletes metadata such as ID / Name

### DISCUSSION THREAD

**Discussion by AndriasScheuchzeri** [marksgraham/OCT-Converter#93] [General]
Hello everyone, 

I have got a question regarding the .fds/.fda files converter.

Is there metadata such as the patient ID/name saved into the derivatives of the .fds/.fda files converter somehow?

If not, the converter would actually work as a bulk pseudonymization tool? 

Any information and comment on this issue are greatly appreciated.

**Comment by marksgraham** [comment]
Hi there,
None of the patient data will be saved out to the files unless you explicitly choose to save it. Just omit the last part about saving metadata from this example and you'll get out anonymised data:

https://github.com/marksgraham/OCT-Converter/blob/main/examples/demo_fda_extraction.py
